pom.xml

```
@@ -24,6 +24,7 @@
 		<!-- DEPENDENCIES VERSION -->
 		<jdk.version>21</jdk.version>
 		<cds.services.version>3.6.0</cds.services.version>
+		<cds-feature-event-hub.version>3.6.0</cds-feature-event-hub.version>
```

```suggestion
		<cds-feature-event-hub.version>${cds.services.version}</cds-feature-event-hub.version>
```
As long it's in sync with cds-services, use this approach.